This isn't referenced by any ManageIQ code or default automate domains

bdunne opened the pull request:
At one point this was an automate dependency for communicating with some external services.  None of the current automate model code and none of the internal code reference this gem.  I think it should be removed and users that need it should maintain their dependencies.

bdunne commented:
@bascar Please review

cc @gmcculloug @mkanoor @tinaafitz @Fryguy @jrafanie

Fryguy commented:
> I think it should be removed and users that need it should maintain their dependencies.

How?

bdunne commented:
How do they manage any of their other dependencies?

gmcculloug commented:
@bdunne It is a manual process today and I think the concern is that this gem is used commonly enough in automation that including it has more benefit then removing it at this time.

Sorry for taking this PR in a different direction, but should we be upgrading to the latest version?  `v2.11`

bdunne commented:
Ok, changed direction.  I created a PR to move it to manageiq-content [here](https://github.com/ManageIQ/manageiq-content/pull/83).  This will have to be merged first, then we can restart tests on manageiq-content.  It fails because I updated the version and it can't `bundle` with this line in the Gemfile.

gmcculloug commented:
Moving to manageiq-content repo makes perfect sense.  👍
